src/test/java/org/arquillian/container/chameleon/ManualSettingsXmlTestCase.java

```diff
@@ -0,0 +1,95 @@
+package org.arquillian.container.chameleon;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.RunWith;
+import org.junit.runner.notification.Failure;
+import org.junit.runners.model.InitializationError;
+
+/**
+ * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
+ */
+public class ManualSettingsXmlTestCase {
```

Comment:
Regarding the thread safety - To which extend should we mark all tests as (not) thread safe...? Why do we have the annotation [ThreadSafe](http://junit.org/junit4/javadoc/4.12/org/junit/runner/notification/RunListener.ThreadSafe.html) then?
Copy-paste from the NotThreadSafe javadoc: 
> it is a bad idea to assume a class is thread-safe without good reason.